src/main/resources/assets/clientcommands/lang/en_us.json

```diff
@@ -42,6 +42,7 @@
 
   "commands.cfindblock.notFound": "No such block found",
   "commands.cfindblock.success.left": "Closest match is at ",
+  "commands.cfindblock.success.glow": " [Glow]",
```

Comment:
Maybe remove the space from here and add it to the code?

Reply:
Sure